build.json

```diff
@@ -38,7 +38,8 @@
             "configuration/roles.json": "roles.d/appkernels.json",
             "configuration/setup.json": "setup.d/appkernels.json",
             "configuration/rest.d": "rest.d",
-            "configuration/assets.d": "assets.d"
+            "configuration/assets.d": "assets.d",
+            "configuration/modules.d/appkernels.json": "modules.d/appkernels.json"
         },
         "etc/crond": {
```

Comment:
Refresh my memory, This places the mapped files into `$INSTALL_ROOT/etc/cron.d`, correct? But the block above it is `etc` and places files into `$INSTALL_ROOT/etc/xdmod`?

Reply:
Not necessarily, files in "etc/crond" are installed to whatever is specified to the `--crondconfdir` option of the `install` script.  If an install root is specified by the `--prefix` option (and `--cronconfdir` is not specified) then those files are installed to `PREFIX/etc/cron.d`.  For an RPM installation the `--crondconfdir` is specified as `%{_sysconfdir}/cron.d` where `%{_sysconfdir}` is typically `/etc`, so those files are placed in `/etc/cron.d`.

Likewise, the "etc" section destination is specified by the `--sysconfdir` option.  For a source install using the `--prefix` option the default is `PREFIX/etc` (not `PREFIX/etc/xdmod`).  For an RPM `%{_sysconfdir}/xdmod` is specified.

Reply:
@jtpalmer Thanks for the clarification